### ISSUE TITLE
subburn(yv12): invalid configuration argument with specific subtitles

### ISSUE THREAD

**Issue by seiya-git** [rigaya/NVEnc#293]
While encoding with subburn filter with this ass subtitle file I always got error
```
subburn: error at subburn(yv12): invalid configuration argument.
Error while running filter "subburn".
```
![изображение](https://user-images.githubusercontent.com/7394323/103892892-64061400-50fd-11eb-8347-4a04a45c1919.png)

sample ass attached
[bad_subtitle.ass](https://github.com/rigaya/NVEnc/files/5781605/bad_subtitle.txt)

**Comment by seiya-git** [comment]
[bad_subtitle.ass.txt](https://github.com/rigaya/NVEnc/files/5787108/bad_subtitle.ass.txt)
one more sub with same error

**Comment by rigaya** [comment]
Thanks for the ass samples, I was able to reproduce the issue. I've made a fix in [NVEnc 5.25](https://github.com/rigaya/NVEnc/releases/tag/5.25).
